### PR TITLE
Updated Meeting Details for sig-usability

### PR DESCRIPTION
**Pull request by hpandeycodeit**
Added Meeting Time and details

### CODE REVIEW

File: sig-usability/README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ The scope of SIG usability is the core end-user usability of the Kubernetes proj
 The [charter](charter.md) defines the scope and governance of the Usability Special Interest Group.
 
 ## Meetings
-* Regular SIG Meeting: [TBDs at TBD PT (Pacific Time)](tbd) (every third week). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=TBD&tz=PT%20%28Pacific%20Time%29).
-  * [Meeting notes and Agenda](https://docs.google.com/document/d/1-dYznZR7Q5XCd2bVYoNfMPYs4uNn1aaePQF0q0JQ0uU/).
+* Regular SIG Meeting: [2 PM (Pacific Time)](https://zoom.us/j/3832562240 ) (every third week). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=TBD&tz=PT%20%28Pacific%20Time%29).
```
Comment (nikhita) [inline sig-usability/README.md]: This needs to be updated in sigs.yaml first and then this file is updated automatically after running `make generate` :)

https://github.com/kubernetes/community/blob/2d82d25e3e6c135ac2b57892fbc2dff005074bc5/sigs.yaml#L2093-L2101

Comment (hpandeycodeit) [reply sig-usability/README.md]: I realized after I pushed the changes. Thanks for reminding :)

**Comment by nikhita** [comment]
@hpandeycodeit Thanks! Can you also squash the commits into a single commit? :grimacing:

File: sigs.yaml [hunk 2]
```diff
@@ -2092,11 +2092,11 @@ sigs:
...
-    day: TBD
-    time: TBD
+    day: Tuesday
+    time: 2 PM
     tz: PT (Pacific Time)
     frequency: every third week
-    url: tbd
+    url: https://zoom.us/j/3832562240
     archive_url: https://docs.google.com/document/d/1-dYznZR7Q5XCd2bVYoNfMPYs4uNn1aaePQF0q0JQ0uU/
     recordings_url: tbd
```
Comment (nikhita) [inline sigs.yaml]: @hpandeycodeit can you also add https://www.youtube.com/playlist?list=PL69nYSiGNLP0eY-U8DVJWHBwKvMDEtOxx here?

File: sigs.yaml [hunk 3]
```diff
@@ -2092,13 +2092,13 @@ sigs:
       company: Lyft
   meetings:
   - description: Regular SIG Meeting
-    day: TBD
-    time: TBD
+    day: Tuesday
+    time: 2 PM
```
Comment (spiffxp) [inline sigs.yaml]: silly style nit, everything else in sigs.yaml uses this format, I'd rather keep us consistent:
```suggestion
    time: "14:00"
```

**Comment by spiffxp** [comment]
/approve
can /lgtm after typo fix (or someone else from SIG can go for it)

File: sigs.yaml (hunk 3 above)
Comment (hpandeycodeit) [reply sigs.yaml]: done :)

**Pull request merged**